ChiModules/LinearBoltzmannSolver/IterativeMethods/lbs_GMRES.cc

```diff
@@ -185,8 +180,8 @@ bool LinearBoltzmann::Solver::GMRES(LBSGroupset& groupset,
       chi_log.ProcessEvent(source_event_tag,
                            ChiLog::EventOperation::AVERAGE_DURATION);
     size_t num_angles = groupset.quadrature->abscissae.size();
-    long int num_unknowns = (long int)glob_dof_count*
-                            (long int)num_angles*
+    long int num_unknowns = (long int)glob_node_count *
```

Comment:
Since I believe these are all size_t, why the casts?

Reply:
Fixed.